app/assets/javascripts/oxalis/store.js

```diff
@@ -201,6 +203,7 @@ export type ReadOnlyTracingType = {
   +boundingBox: ?BoundingBoxType,
   +restrictions: RestrictionsType & SettingsType,
   +isPublic: boolean,
+  +tags: Array<string>,
```

Comment:
We should really refactor these types. BoundingBox, restrictions, isPublic, tags (and probably more) seem to exist in all tracing types.